src/plugin/parse-tika/src/java/org/apache/nutch/parse/tika/TikaParser.java

```diff
@@ -109,7 +114,18 @@ public Parse getParse(String url, WebPage page) {
     HTMLDocumentImpl doc = new HTMLDocumentImpl();
     doc.setErrorChecking(false);
     DocumentFragment root = doc.createDocumentFragment();
-    DOMBuilder domhandler = new DOMBuilder(doc, root);
+   // DOMBuilder domhandler = new DOMBuilder(doc, root);
+    ContentHandler domHandler;
+    // Check whether to use Tika's BoilerplateContentHandler
+    if (useBoilerpipe) {
+        LOG.debug("Using Tikas's Boilerpipe with Extractor: " + boilerpipeExtractorName);
```

Comment:
Can also use more efficient slf4j convention
logger.debug("The entry is {}.", entry);